pallets/author-mapping/src/lib.rs

```diff
@@ -70,6 +71,18 @@ pub mod pallet {
 			write!(f, "{:?}{:?}", self.0, self.1)
 		}
 	}
+	impl<T: Config> TryFrom<Vec<u8>> for KeysWrapper<T> {
+		type Error = Error<T>;
+		fn try_from(input: Vec<u8>) -> Result<Self, Self::Error> {
```

Comment:
It might be better to take a slice here instead of a vec

Reply:
Why that ?